modules/ossm-federation-create-import.adoc

```diff
@@ -0,0 +1,112 @@
+////
+This module included in the following assemblies:
+* service_mesh/v2x/ossm-federation.adoc
+////
+
+[id="ossm-federation-create-import_{context}"]
+= Creating an ImportedServiceSet
+
+You create an `ImportServiceSet` resource to explicitly declare the services that you want to import into your mesh.
+
+Services are imported as `<exported-name>.<exported-namespace>.svc.<peer-name>.remote` which is a "hidden" service, visible only within the egress gateway namespace, along with the "real" service accessible to the mesh ``<export-name>.<export-namespace>.svc.<local-federation-name>.local`. You can treat the local import just like any other service in the mesh. It automatically routes through the egress gateway, where it is redirected to the exported service's remote name.
```

Comment:
```suggestion
Services are imported with the name `<exported-name>.<exported-namespace>.svc.<ServiceMeshPeer.name>.remote` which is a "hidden" service, visible only within the egress gateway namespace and is associated with the exported service's hostname (link to "ossm-federation-create-export_{context}"). The service will be available locally as `<export-name>.<export-namespace>.<domainSuffix>`, where `domainSuffix` is `svc.<ServiceMeshPeer.name>-imports.local` by default, unless `importAsLocal` is `true`, in which case `domainSuffix` is `svc.cluster.local`, otherwise, the domain suffix in the import rule will be applied.  You can treat the local import just like any other service in the mesh. It automatically routes through the egress gateway, where it is redirected to the exported service's remote name.
```